### PR TITLE
Source Import API

### PR DESCRIPTION
**Pull request by brix4dayz**
- `PUT` and `GET` for `/repos/:owner/:repo/import` according to the [preview docs](https://developer.github.com/v3/migration/source_imports/)
- Recycled from #505

### CODE REVIEW

File: github/SourceImport.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
+import github.GithubObject
```
Comment (jasonwhite) [inline github/SourceImport.py]: This file is missing the Python source encoding comment (`# -*- coding: utf-8 -*-`) and copyright header.

File: github/tests/SourceImport.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (jasonwhite) [inline github/tests/SourceImport.py]: Same with this file.

File: github/SourceImport.py [hunk 3]
```diff
@@ -0,0 +1,190 @@
+
+# -*- coding: utf-8 -*-
```
Comment (jasonwhite) [inline github/SourceImport.py]: I believe this needs to be the first line in the file.

**Comment by sfdye** [comment]
Thank you very much for your contribution @brix4dayz

**Pull request merged**